packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -181,6 +181,26 @@ function flattenTree( input = {}, prefix, token ) {
 	return result;
 }
 
+/**
+ * Resolves ref into the value it is pointing to.
+ *
+ * @param {Object} refObj      The reference we are pointing to.
+ *
+ * @param {string} blockStyles Block styles.
```

Comment:
```suggestion
 * @param {string} blockStyles The tree of block styles.
```

The argument passed in is `tree` so maybe we should retain that knowledge of shape here?